censys/certificates.py

```diff
@@ -2,53 +2,152 @@
+                },
+            },
+            "__expanded_names": ["google.com", "com", "ns1.google.com"],
+            "serial_number": "1952809538",
+            "validity": {
+                "start": "2015-09-10T06:49:19Z",
+                "length": 31536000,
+                "end": "2016-09-09T06:49:19Z",
+            },
+            "issuer_dn": "CN=ns1.google.com, emailAddress=ssl@ns1.google.com",
+            "fingerprint_sha1": "3704a663ed5db89f85c5377852d678b299389775",
+            "version": 3,
+            "fingerprint_sha256": "fce621c0dc1c666d03d660472f636ce91e66e96460545f0da7eb1a24873e2f70",
+            "names": ["ns1.google.com"],
+            "tbs_fingerprint": "e9bae1d0af98bfebcb73d061e2b5088d33ecf4e29646b5afbcc9b0026f9f8641",
+            "validation_level": "unknown",
+            "extensions": {
+                "authority_key_id": "dc3e2e3f49b126782f2dd4444c3cb5970be85979",
+                "subject_key_id": "dc3e2e3f49b126782f2dd4444c3cb5970be85979",
+                "basic_constraints": {"is_ca": False},
+            },
+            "fingerprint_md5": "37ec912bf30802fd4ab3592f8713ef6f",
+            "subject_key_info": {
+                "fingerprint_sha256": "5cbcb9740f3d8a447d4267722092ad405701c6036d93b1a84c512ebefbcbe346",
+                "key_algorithm": {"name": "RSA"},
+                "rsa_public_key": {
+                    "length": 2048,
+                    "modulus": "vYjm5ZGIdEk1Wo7bodnJsiKNiqPZoEsUXqoDV+TGpIxCZKkGB3oJscoHaDYubVBUFuuc+6/TtD371hEWI8aMhPA/QhRBtZuKDXUygIgHy18SBG2ciJb7xcRIBdO9FNmspflxFX5oLPbVc+LjXPz5Cs1J3dASqh+QDWjddvmMaRg6U+oQHmROYq3bo+NjlF+PFc6XJ+zug9VDEppNKhvgw83YFOaS+olbKLus34TxW2C17Q0q8ABasn3GDfp+O5ixZHkfMJU1grsrQZR29oHkgC54NytOFhiWjLoSslYY5iFZRPnDr4GtGZzXmrZXPwwiNUqI2oauZ8m02tayCEPfLw==",
+                    "exponent": 65537,
+                },
+            },
+            "signature_algorithm": {
+                "oid": "1.2.840.113549.1.1.11",
+                "name": "SHA256-RSA",
+            },
+            "spki_subject_fingerprint": "28359cf3792cf85e8aca5cb9f3eaf6b37c1784892b9782100e8df47568201af8",
+            "issuer": {
+                "common_name": ["ns1.google.com"],
+                "email_address": ["ssl@ns1.google.com"],
+            },
+        },
+        "metadata": {
+            "post_processed": False,
+            "parse_status": "success",
+            "updated_at": "2018-08-31T14:06:01",
+            "source": "scan",
+            "seen_in_scan": True,
+            "parse_version": 0,
+        },
+    }
 
     @classmethod
     def setUpClass(cls):
         cls._api = CensysCertificates()
 
     def testGet(self):
-        self.assertEqual(self._api.view("fce621c0dc1c666d03d660472f636ce91e66e96460545f0da7eb1a24873e2f70"),
-                         self.correct_get_result)
+        self.assertEqual(
+            self._api.view("fce621c0dc1c666d03d660472f636ce91e66e96460545f0da7eb1a24873e2f70"),
+            self.correct_get_result,
```

Comment:
Rather than try to ensure the result structure is always up to date, I would check that the fingerprints are matched, and that the returned object is a dictionary.